spec/index.html

```diff
@@ -545,6 +545,11 @@ <h3>RDF Version Announcement</h3>
 Host: example.com
 Accept: text/turtle; version=1.2
     </pre>
+    <p>
+      Defined <a>version labels</a> to be used with the <code>version</code> parameter 
+      and in <a>concrete RDF syntax</a> are given in the
+      <a href="#defined-version-labels">section below</a>.
```

Comment:
I had to read this sentence three times to understand what it wants to say, which is due to its use of passive voice and the fact that it begins with the object rather than the subject. I propose to rephrase it as follows.
```suggestion
      Section <a href="#defined-version-labels" class="sectionRef"></a> defines <a>version labels</a>
      to be used with the <code>version</code> parameter and in <a>concrete RDF syntax</a>.
```